torch/_ops.py

```diff
@@ -25,6 +27,22 @@ def dl_open_guard():
     if _SET_GLOBAL_FLAGS:
         sys.setdlopenflags(old_flags)
 
+def get_tags():
+    tags = {}
+    native_yaml_path = os.path.join('aten/src/ATen/native/native_functions.yaml')
+    tags_yaml_path = os.path.join('aten/src/ATen/native/tags.yaml')
```

Comment:
This isn't going to fly, pretty sure we don't ship these files in binary distribution.